readme.md

```diff
@@ -1 +1,54 @@
-readme
\ No newline at end of file
+**F**irefo**x** **A**ccounts **P**age **O**bject **M**odel
+===============================================
+Webdriver compatible page object model and utilities for Firefox Accounts
+
+Overview
+-------------
+This package contains a utility to create a test Firefox Account on either the dev or prod instance of Firefox Accounts,
+as well as a set of page objects that can be used to interact with Firefox Accounts' sign in screens.
+
+Usage
+-----
+To create a test Firefox Account, use the `create_account` method in the `FxATestAccount` object.
+You should pass the base url for the site for which you are creating the account into the constructor
+for `FxATestAccount`, or, if you know you want to create a production Account, you can omit that argument.
+
+Example:
+```
+from fxapom.fxapom import FxATestAccount
+acct = FxATestAccount(base_url='https://www-dev.allizom.org').create_account()
+```
+
+To sign in via Firefox Accounts, use the `sign_in` method in the `WebDriverFxA` object,
+passing in the email addresss and password.
+
+Example:
+```
+from fxapom.fxapom import WebDriverFxA
+fxa = WebDriverFxA(mozwebqa)
```

Comment:
We should probably note that using this will only work with pytest-mozwebqa. It might be nice to support passing in the selenium instance and optional timeout for anyone not using our plugin.